dev-tools/mage/settings.go

```diff
@@ -41,6 +41,8 @@ const (
 
 	elasticAgentModulePath = "github.com/elastic/elastic-agent"
 
+	defaultBeatName = "elastic-agent"
```

Comment:
```suggestion
	defaultName = "elastic-agent"
```

If we are going to change it, at least don't reference Beats anymore.